packages/components/organisms/f-checkout/src/components/Accordion.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div>
+        <button
+            :id="`${id}-accordion-header`"
+            :aria-controls="`${id}-header`"
+            :aria-expanded="isExpanded"
+            :class="$style['c-checkout-accordion-header']"
+            :data-test-id="`${id}-accordion-header`"
+            type="button"
+            @click="toggle">
+            {{ title }}
+            <chevron-icon
+                :class="[
+                    $style['c-checkout-accordion-header-arrow'], {
+                        [$style['c-checkout-accordion-header-arrow--expanded']]: isExpanded
+                    }]" />
+        </button>
+        <section
+            :id="`${id}-accordion-section`"
+            :aria-labelledby="`${id}-section`"
+            :class="[
+                $style['c-checkout-accordion-section'], {
+                    [$style['c-checkout-accordion-section--hidden']]: !isExpanded
+                }]"
+        >
+            <slot />
+        </section>
+    </div>
+</template>
+
+<script>
+import { ChevronIcon } from '@justeat/f-vue-icons';
+
+export default {
+    components: {
+        ChevronIcon
+    },
+
+    props: {
+        id: {
+            type: String,
+            required: true
+        },
+        title: {
+            type: String,
+            required: true
+        }
+    },
+
+    data () {
+        return {
+            isExpanded: false
+        };
+    },
+
+    methods: {
+        toggle () {
+            this.isExpanded = !this.isExpanded;
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+    $icon-size: 16px;
```

Comment:
`spacing(x2)`

Reply:
Not sure I agree with this one. In theory the icon sizes aren't related to the spacing. If this value is repeated then we could have a separate variable for icon sizes.